src/integrationTest/java/uk/gov/hmcts/darts/event/controller/EventsControllerCourtLogsTest.java

```diff
@@ -120,4 +146,54 @@ private List<EventEntity> getAllLogEventsMatchingText() {
             .toList();
     }
 
+    @Test
+    void courtLogsGet() throws Exception {
+
+        MockHttpServletRequestBuilder requestBuilder = get(ENDPOINT)
+            .queryParam("Courthouse", "Swansea")
+            .queryParam("caseNumber", CASE_0000001)
+            .queryParam("startDateTime", String.valueOf(CommonTestDataUtil.createOffsetDateTime("2022-07-01T09:00:00")))
+            .queryParam("endDateTime", String.valueOf(CommonTestDataUtil.createOffsetDateTime("2022-07-01T11:00:00")))
+            .contentType(MediaType.APPLICATION_JSON_VALUE);
+        mockMvc.perform(requestBuilder).andExpect(MockMvcResultMatchers.status().isOk());
+
+    }
+
+    @Test
+    void courtLogsGetNoParametersPassed() throws Exception {
+
+        MockHttpServletRequestBuilder requestBuilder = get(ENDPOINT).contentType(MediaType.APPLICATION_JSON_VALUE);
+
+        mockMvc.perform(requestBuilder).andExpect(MockMvcResultMatchers.status().isBadRequest()).andExpect(
+            MockMvcResultMatchers.header().string(
+                "Content-Type",
+                "application/problem+json"
+            ));
+    }
+
+    @Test
+    void courtLogsGetResultMatch() throws Exception {
```

Comment:
Could we have a test whereby there are 2 events for the case we are interested in, and 2 events for a different case on the same day in the same courtroom, to make sure we only get back the 2 events linked to the case we are interested in?